<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="1" total="2" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="25532" started-at="2020-01-02T18:38:20Z" finished-at="2020-01-02T18:38:46Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="25532" started-at="2020-01-02T18:38:20Z" finished-at="2020-01-02T18:38:46Z">
      <class name="testcases.LoginTestActiTime">
        <test-method status="PASS" signature="setUp()[pri:0, instance:testcases.LoginTestActiTime@9660f4e]" name="setUp" is-config="true" duration-ms="15226" started-at="2020-01-03T00:08:20Z" finished-at="2020-01-03T00:08:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="pageLoad()[pri:0, instance:testcases.LoginTestActiTime@9660f4e]" name="pageLoad" is-config="true" duration-ms="251" started-at="2020-01-03T00:08:36Z" finished-at="2020-01-03T00:08:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- pageLoad -->
        <test-method status="FAIL" signature="invalidLoginApp()[pri:1, instance:testcases.LoginTestActiTime@9660f4e]" name="invalidLoginApp" duration-ms="34" started-at="2020-01-03T00:08:36Z" finished-at="2020-01-03T00:08:36Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at testcases.LoginTestActiTime.invalidLoginApp(LoginTestActiTime.java:40)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:823)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:778)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1225)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1150)
	at org.testng.TestNG.runSuites(TestNG.java:1075)
	at org.testng.TestNG.run(TestNG.java:1047)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalidLoginApp -->
        <test-method status="FAIL" signature="takePhoto(org.testng.ITestResult)[pri:0, instance:testcases.LoginTestActiTime@9660f4e]" name="takePhoto" is-config="true" duration-ms="0" started-at="2020-01-03T00:08:36Z" finished-at="2020-01-03T00:08:36Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=invalidLoginApp status=FAILURE method=LoginTestActiTime.invalidLoginApp()[pri:1, instance:testcases.LoginTestActiTime@9660f4e] output={null}]]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at generic.BaseClass.takePhoto(BaseClass.java:71)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:216)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:712)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:823)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:778)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1225)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1150)
	at org.testng.TestNG.runSuites(TestNG.java:1075)
	at org.testng.TestNG.run(TestNG.java:1047)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takePhoto -->
        <test-method status="SKIP" signature="pageLoad()[pri:0, instance:testcases.LoginTestActiTime@9660f4e]" name="pageLoad" is-config="true" duration-ms="0" started-at="2020-01-03T00:08:45Z" finished-at="2020-01-03T00:08:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- pageLoad -->
        <test-method status="SKIP" signature="validLoginApp()[pri:1, instance:testcases.LoginTestActiTime@9660f4e]" name="validLoginApp" duration-ms="15" started-at="2020-01-03T00:08:45Z" finished-at="2020-01-03T00:08:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validLoginApp -->
        <test-method status="FAIL" signature="takePhoto(org.testng.ITestResult)[pri:0, instance:testcases.LoginTestActiTime@9660f4e]" name="takePhoto" is-config="true" duration-ms="0" started-at="2020-01-03T00:08:45Z" finished-at="2020-01-03T00:08:45Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=validLoginApp status=SKIP method=LoginTestActiTime.validLoginApp()[pri:1, instance:testcases.LoginTestActiTime@9660f4e] output={null}]]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at generic.BaseClass.takePhoto(BaseClass.java:82)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:216)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:712)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:823)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:778)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1225)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1150)
	at org.testng.TestNG.runSuites(TestNG.java:1075)
	at org.testng.TestNG.run(TestNG.java:1047)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takePhoto -->
        <test-method status="SKIP" signature="tearDown()[pri:0, instance:testcases.LoginTestActiTime@9660f4e]" name="tearDown" is-config="true" duration-ms="0" started-at="2020-01-03T00:08:46Z" finished-at="2020-01-03T00:08:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testcases.LoginTestActiTime -->
      <class name="generic.BaseClass">
        <test-method status="PASS" signature="setUpSuite()[pri:0, instance:generic.BaseClass@7c29daf3]" name="setUpSuite" is-config="true" duration-ms="1087" started-at="2020-01-03T00:08:19Z" finished-at="2020-01-03T00:08:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpSuite -->
      </class> <!-- generic.BaseClass -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
